android/KMEA/app/src/main/assets/keyboard.html

```diff
@@ -145,9 +145,10 @@
       kmw['osk']['show'](true);
     }
 
-    function insertText(dn, s) {
-      //window.console.log('insertText('+ dn +', ' + s +');');
-      window.jsInterface.insertText(dn, s);
+    function insertText(dn, s, dr) {
```

Comment:
Since we are changing these definitions, let's go ahead and add jsdoc comments so that we know what `dn` and `dr` in the future :)